src/library/scala/util/Using.scala

```diff
@@ -372,14 +372,5 @@ object Using {
     *
     * @tparam R the type of the resource
     */
-  trait Releasable[-R] {
-    /** Releases the specified resource. */
-    def release(resource: R): Unit
-  }
-
-  object Releasable {
-    /** An implicit `Releasable` for [[java.lang.AutoCloseable `AutoCloseable`s]]. */
-    implicit val autoCloseableIsReleasable: Releasable[AutoCloseable] = (resource: AutoCloseable) => resource.close()
-  }
-
+  type Releasable[-R] = AutoCloseable[R]
```

Comment:
Going one step further would be to remove this alias altogether, and change `T: Releasable` to just be `T <: AutoCloseable` everywhere in `Using.scala`. Either way it won't have any extra indirection which is a good thing.

Reply:
@smarter You beat me to it, but I'd in favor of removing the alias altogether, especially given that it doesn't seem to "just work" out of the box based on the CI output.